doc/user/setup.rst

```diff
@@ -41,7 +41,7 @@ Daemons Configuration File
 --------------------------
 There is another file that controls the default options passed to daemons when
 starting FRR as a service. This file is located in your configuration
-directory, usually at :file:`/etc/frr/daemons`.
+directory, usually at :file:`/etc/frr/daemons.conf`.
```

Comment:
`/etc/frr/daemons` is actually the correct path for future versions. Probably the most helpful way to change this would be to say
```
usually at :file:`/etc/frr/daemons` or :file:`/etc/frr/daemons.conf`
```